.github/scripts/scrape-translations.sh

```diff
@@ -11,6 +11,7 @@ gtregex="getTranslation\(\s*?[\"'](.*?)[\"'],.*?[\"'](.*?)[\"'].*?\)"
 directory=${1-./packages}
 
 # Get all registered enums if no directory is specified
+teoutput=""
```

Comment:
Make sure this is always defined. Otherwise it can be undefined below when a directory is set.